src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -139,9 +125,12 @@ public static void main(final String[] args) {
       usage();
       return;
     }
-    SwingUtilities.invokeLater(LookAndFeel::setupLookAndFeel);
 
-    HttpProxy.setupProxies();
+    LookAndFeel.setupLookAndFeel();
```

Comment:
I did notice two of these. I do believe it needs to be called before the Swing components are created.

The significant UI change is likely the UI theme default to the system default. That was updated intentionally to avoid special casing Mac OS behavior. I'm open to updating the default back to what it was, but note we had an OS specific setting there.